code/segmentedtickerplant/datastripe.q

```diff
@@ -1,60 +1,64 @@
-//configload loads configurable .csv files into memory so relevant filters can be applied to data.
-//loads segmenting.csv which defines how many segments there will be.
-//loads filtermap.csv which defines what filters should be applied to each segment.
+//Allows config file to be overwritten in process.csv
 
-//.ds.segmentconfig/segmentfiltermap are variables which can take different csv files. These files can be chosen on in process.csv.
-//error trap these .ds variables incase this file is loaded stand alone.
+.ds.stripeconfig:@[value;`.ds.stripeconfig;`striping.json];
 
-.ds.segmentconfig:@[value;`.ds.segmentconfig;`segmenting.csv];
-.ds.filtermap:@[value;`.ds.filtermap;`filtermap.csv];
-
-//if statement checks segmenting.csv and filtermap.csv exist. If not, process exited and message sent to error logs.
-//configload transfoms segmenting.csv into table format so it can be accessed.
-//configload transforms filtermap.csv into table format then into a mapping of wcRef to filter which can be accessed and applied to data.
-//checks csv files load correctly. If not, process exited and message sent to error logs.
+//Loads the striping.json config file checks if each subscriptiondefault is set for each segment and errors if not defined
 
 configload:{
-     scpath:first .proc.getconfigfile[string .ds.segmentconfig];
-     fmpath:first .proc.getconfigfile[string .ds.filtermap];
-     {if[()~key hsym x;.lg.e[`init;"The following file can not be found: ",string x]]} each (scpath;fmpath);
-     @[{.stpps.segmentconfig:("SIS";enlist",")0: hsym x};scpath;{.lg.e[`init;"Failure in loading ",string y]}[;scpath]];
-     @[{.stpps.segmentfiltermap:(!/)(("S*";enlist",")0: hsym x)`wcRef`filter};fmpath;{.lg.e[`init;"Failure in loading ",string y]}[;fmpath]];
+     scpath:first .proc.getconfigfile[string .ds.stripeconfig];
+     {if[()~key hsym x;.lg.e[`init;"The following file can not be found: ",string scpath]]};
+     .stpps.stripeconfig:.j.k raze read0 scpath;
```

Comment:
```suggestion
     .stpps.stripeconfig:.j.k read1 scpath;
```

you can skip the `raze` if you read the file as bytes with `read1` & `.j.k` will still parse it